src/components/Menu.tsx

```diff
@@ -1,92 +1,170 @@
+		},
+		{
+			link: '/guides',
+			label: 'guides',
+			hideOnHeader: false,
+			subMenu: (navDocs && navDocs.label === 'guides' && navDocs) || null,
+		},
+		{
+			link: '/governance',
+			label: 'governance',
+			hideOnHeader: false,
+		},
+		{
+			externalLink: 'https://discord.com/invite/AEdUHzt',
+			label: 'community',
+			hideOnHeader: false,
+		},
+		{
+			externalLink: 'https://jobs.defialliance.co/companies/synthetix',
+			label: 'careers',
+			hideOnHeader: true,
+		},
+		{
+			externalLink: 'https://blog.synthetix.io/',
+			label: 'blog',
+			hideOnHeader: false,
+		},
+		{
+			externalLink: 'https://research.synthetix.io/',
+			label: 'research',
+			hideOnHeader: true,
+		},
+	];
+
+	const router = useRouter();
+	const urlFolderPathName = router.pathname.split('/')[1];
+
 	return (
-		<StyledMenu isOpen={!!isOpen} {...rest}>
-			{data.map(
-				(item) =>
-					(!isHeader || !item.hideOnHeader) && (
-						<MenuItem isHeader={!!isHeader} key={item.label}>
-							{item.link ? (
-								<Link href={item.link}>
-									<a>{item.label}</a>
-								</Link>
-							) : (
-								<ExternalLink href={item.externalLink} key={item.label}>
-									{item.label}
-								</ExternalLink>
-							)}
-						</MenuItem>
-					)
-			)}
-		</StyledMenu>
+		<>
+			<StyledMenu isOpen={!!isOpen} {...rest}>
+				{data.map((item) => {
+					if (isHeader) {
+						return (
+							!item.hideOnHeader &&
+							!item.button && (
+								<MenuItem key={item.label} subOpen={!!subOpen} {...rest}>
+									{item.link ? (
+										<Link href={item.link}>
+											<a className={urlFolderPathName === item.label ? 'active' : ''}>
+												{item.label}
+											</a>
+										</Link>
+									) : (
+										<ExternalLink href={item.externalLink} key={item.label}>
+											{item.label}
+										</ExternalLink>
+									)}
+								</MenuItem>
+							)
+						);
+					} else {
+						return (
+							<MenuItem subOpen={false} key={item.label}>
+								{item.link ? (
+									<Link href={item.link}>
+										<a>{item.label}</a>
+									</Link>
+								) : (
+									<ExternalLink href={item.externalLink}>{item.label}</ExternalLink>
+								)}
+							</MenuItem>
+						);
+					}
+				})}
+
+				{data.map((item) => {
+					if (isHeader) {
+						return (
+							item.button && (
+								<MenuBtn
+									key={item.label}
+									border={item.label === 'staking' ? true : false}
+									subOpen={!!subOpen}
+									{...rest}
+								>
+									{item.link ? (
+										<Link href={item.link}>
+											<a>
+												{item.label}
+												<span>
+													<ImArrowUpRight2 />
+												</span>
+											</a>
+										</Link>
+									) : (
+										<ExternalLink href={item.externalLink} key={item.label}>
+											{item.label}{' '}
+											<span>
+												<ImArrowUpRight2 />
+											</span>
+										</ExternalLink>
+									)}
+								</MenuBtn>
+							)
+						);
+					}
+				})}
```

Comment:
the second loop looks redundant, can be added to the first loop and rendered there